core/src/test/java/io/grpc/transport/MessageFramerTest.java

```diff
@@ -270,5 +270,10 @@ public int hashCode() {
     public WritableBuffer allocate(int capacityHint) {
       return new ByteWritableBuffer(Math.min(maxSize, Math.max(capacityHint, minSize)));
     }
+
+    @Override
+    public WritableBuffer empty() {
+      return new ByteWritableBuffer(0);
```

Comment:
I still think we should just make this a singleton.  Is there any reason not to?